lib/Dialect/SPIRV/SPIRVDialect.cpp

```diff
@@ -150,7 +162,8 @@ static Type parseAndVerifyType(SPIRVDialect const &dialect, StringRef spec,
 //                | vector-type
 //                | spirv-type
 //
-// array-type ::= `!spv.array<` integer-literal `x` element-type `>`
+// array-type ::= `!spv.array<` integer-literal `x` element-type
+// (`[` integer-literal `]`)? `>`
```

Comment:
Nit : Please indent the expression to align with `!spv.array`

Reply:
Thanks.